consensus/core/Cargo.toml

```
@@ -31,7 +31,7 @@ mockall.workspace = true
 mysten-common.workspace = true
 mysten-metrics.workspace = true
 mysten-network.workspace = true
-nom = "7.1.3"
+nom = { workspace = true }
```

If the dependency just has `workspace` attribute, can you flatten it?

```suggestion
nom.workspace = true
```